packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -130,6 +127,15 @@ class ScrollableBaseNative extends Component {
 		 */
 		focusableScrollbar: PropTypes.bool,
 
+		/**
+		 * Prevents scroll by dragging or flicking on the list or the scroller.
+		 *
+		 * @type {Boolean}
+		 * @default false
```

Comment:
The default value for `Moonstone` is not `false` but `true`.

Reply:
It is the origin of my question above. Please see my comment.

Reply:
I updated the name and its default value is now `false`.